src/helper/network/proto.cpp

```diff
@@ -17,13 +17,14 @@ using dds::network::request_init;
 using dds::network::request_shutdown;
 
 namespace {
+// NOLINTNEXTLINE(cert-err58-cpp,fuchsia-statically-constructed-objects)
+const std::map<std::string_view, request_id> mapping = {
+    {client_init::request::name, client_init::request::id},
+    {request_init::request::name, request_init::request::id},
+    {request_shutdown::request::name, request_shutdown::request::id}};
```

Comment:
It seems as though when this object is a function local static, it is freed before the fuzzer exit handler (atexit), causing a use-after-free.